interface/forms/group_attendance/functions.php

```diff
@@ -138,7 +138,7 @@ function insert_patient_encounter($pid, $gid, $group_encounter_date, $participan
     else{
         $insert_encounter_sql =
             "INSERT INTO form_encounter (date, reason, pid, encounter, pc_catid, provider_id, external_id) ".
-            "VALUES (?, ?, ?, ?, 1000, ?, ?);";
+            "VALUES (?, ?, ?, ?, 15, ?, ?);";
```

Comment:
should not hard-code 15 (for now would just insert the pc_catid of the lowest id you get when query 'SELECT pc_catid FROM openemr_postcalendar_categories WHERE pc_cattype = 3 AND pc_active = 1' )